_episodes/06-testing.md

```diff
@@ -23,16 +23,17 @@ keypoints:
 >  - How to run Docker container with CI platform?
 
 
-This lesson extends the software testing subject 
-and introduces Continuous Integration workflow. Examples of Python testing frameworks 
-and CI platforms will be used.   
+This lesson extends the software testing subject and introduces Continuous Integration 
+workflow. 
+Examples of Python testing frameworks and CI platforms will be used.   
 
 ### Lesson outline
 
 - Element 1: Unit, integration and regression tests
 - Element 2: Python testing frameworks
-- Element 3: Continuous Integration practice
-- Element 4: Continuous Integration Service: Travis CI
+- Element 3: Regression tests for Simple Workflow
+- Element 4: Continuous Integration practice
+- Element 5: Continuous Integration Service: Travis CI
 
 
 ### Lesson requirements
```

Comment:
i would simply point to the software carpentry unit tests here. no need to repeat the basics of unit tests here. part of the goal here is to focus on imaging researchers/developers more than the general audience.